ansible/roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -201,6 +213,30 @@ resource "openstack_networking_port_v2" "login" {
   }
 }
 
+# Storage VLAN
+{% if cluster_storage_network is defined %}
+resource "openstack_networking_port_v2" "login_storage" {
+  name           = "{{ cluster_name }}-login-storage-0"
+  network_id     = data.openstack_networking_network_v2.cluster_storage_direct.id
+  admin_state_up = "true"
+
+  # not sure if needed here
```

Comment:
I think we need it, for the moment.

We might have to eventually make this option, when we test with VF-LAG, etc.